### PR TITLE
Modal sizing

### PR DESCRIPTION
**Pull request by guyesh**
**Edited the modal size on different screens:**
 on short screens - made it high,
on high screens - made it short.
on high and thin screens (mostly phones) - fill almost all screen (with a gap).
**Edited the modal scrollbar style to fit the main page.**

### CODE REVIEW

**Comment by guyesh** [comment]
VID-20201013-164254
*does not show the new scrollbar style  only responsive modal sizes

**Review by uriklar — APPROVED** [review]

File: src/components/Modal/Modal.js [hunk 1]
```diff
@@ -119,25 +119,52 @@ const ModalWrapper = styled(ReactModal)`
     left: 25px;
     right: 25px;
     bottom: 20px;
-    max-height: calc(100vh - 50px);
+    max-height: calc(100vh - 50px) !important;
```
Comment (guytepper) [inline src/components/Modal/Modal.js]: Why is it needed? 
Seems like it's already declared in [line 110](https://github.com/guytepper/1km.co.il/pull/136/files#diff-869a02a1420cc0268fa726a77b96fea2f08e3bc575b1398af4af25bea39694f5R110)

File: src/components/Modal/Modal.js [hunk 2]
```diff
@@ -119,25 +119,52 @@ const ModalWrapper = styled(ReactModal)`
...
+  }
+  /* in high screens, modal is shorter (only if its not a phone - width is large)*/
+  @media (min-height: 850px) and (min-width: 1280px) {
+    max-height: calc(100vh - 350px) !important;
+  }
+
+  /*scrollbar*/
+  scrollbar-color: #5f6ffa #dde0ff;
+  scrollbar-width: thin;
+  ::-webkit-scrollbar {
```
Comment (guytepper) [inline src/components/Modal/Modal.js:152]: Add space between variables & style declaration.

```suggestion

  ::-webkit-scrollbar {
```

File: src/components/Modal/Modal.js [hunk 3]
```diff
@@ -119,25 +119,52 @@ const ModalWrapper = styled(ReactModal)`
...
+    max-height: calc(100vh) !important;
+  }
+  /* in high screens, modal is shorter (only if its not a phone - width is large)*/
+  @media (min-height: 850px) and (min-width: 1280px) {
+    max-height: calc(100vh - 350px) !important;
+  }
+
+  /*scrollbar*/
+  scrollbar-color: #5f6ffa #dde0ff;
+  scrollbar-width: thin;
```
Comment (guytepper) [inline src/components/Modal/Modal.js:150]: Is it possible to make it clearer that those are variables?

File: src/components/Modal/Modal.js [hunk 4]
```diff
@@ -119,25 +119,52 @@ const ModalWrapper = styled(ReactModal)`
...
+  @media (max-height: 700px) {
+    top: 30px !important;
+    max-height: calc(100vh) !important;
+  }
+  /* in high screens, modal is shorter (only if its not a phone - width is large)*/
+  @media (min-height: 850px) and (min-width: 1280px) {
+    max-height: calc(100vh - 350px) !important;
+  }
+
+  /*scrollbar*/
```
Comment (guytepper) [inline src/components/Modal/Modal.js]: Let's make the section comments more neat:
```suggestion
//**** Scrollbar ****//
```

File: src/components/Modal/Modal.js [hunk 5]
```diff
@@ -119,25 +119,52 @@ const ModalWrapper = styled(ReactModal)`
...
   }
 
   @media (min-width: 1440px) {
     top: 100px;
     left: 500px;
     right: 500px;
+    max-height: calc(100vh - 200px);
+  }
+  /*in short screens - modal higher*/
+  @media (max-height: 700px) {
```
Comment (guytepper) [inline src/components/Modal/Modal.js]: It seems here that you are chose to 'fight' against the styles that were already declared before. Why not change them directly?
You can go to the initial styles (from line `108` and below) or to the media queries (such as `min-width: 360px`) and put/exchange those styles there.

File: src/components/Modal/Modal.js [hunk 6]
```diff
@@ -119,25 +119,52 @@ const ModalWrapper = styled(ReactModal)`
...
+    max-height: calc(100vh - 350px) !important;
+  }
+
+  /*scrollbar*/
+  scrollbar-color: #5f6ffa #dde0ff;
+  scrollbar-width: thin;
+  ::-webkit-scrollbar {
+    width: 5px;
+  }
+  ::-webkit-scrollbar-track {
```
Comment (guytepper) [inline src/components/Modal/Modal.js:156]: Add space
```suggestion

  ::-webkit-scrollbar-track {
```

File: src/components/Modal/Modal.js [hunk 7]
```diff
@@ -119,25 +119,52 @@ const ModalWrapper = styled(ReactModal)`
...
+    background: #dde0ff;
+    border-radius: 10px;
+  }
+  ::-webkit-scrollbar-thumb {
```
Comment (guytepper) [inline src/components/Modal/Modal.js:161]:
```suggestion

  ::-webkit-scrollbar-thumb {
```

**Review by guytepper — CHANGES_REQUESTED** [review]
Thanks! Check my comments and talk to me if something's not clear.

Also, check this scenario (`1270x1020`): 
Screen Shot 2020-10-14 at 17 58 08

**Pull request merged**